stablehlo/dialect/StablehloOps.td

```diff
@@ -1862,11 +1863,11 @@ def StableHLO_BroadcastInDimOp : StableHLO_Op<"broadcast_in_dim",
     ```
   }];
   let arguments = (ins
-    HLO_Tensor:$operand /*broadcast_in_dim_i1*/,
+    HLO_TensorOrPerAxisQuantizedTensor:$operand /*broadcast_in_dim_i1*/,
```

Comment:
Can we make similar changes in `broadcast` as well?

Reply:
Done, I left it untouched after reading a comment -  on its way out of StableHLO